src/calibrate/requirements/prod.txt

```diff
@@ -0,0 +1,12 @@
+Flask==1.1.2
```

Comment:
These two requirement files (dev.txt, prod.txt) are duplicate. Probably we could stick with one for easy management.

Reply:
Makes sense - I was following the format of other modules. Perhaps we could change this later. @ja-odur - please let me know if you require a change for approval.

Reply:
Yes, we could fix this later

Reply:
Great. Are you happy to approve @ja-odur?

Reply:
I will change to a single `requirements.txt` file.